x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -121,6 +145,40 @@ export const ActionTypeForm = ({
     return defaultParams;
   };
 
+  const getBoundThrottle = useCallback(
+    (throttle: number | null, throttleUnit: string) => {
```

Comment:
I think this is to prevent users to select a throttle interval lesser than the check interval.
This works but does not give any feedback to the user, it just sets the value same as the check interval.
Maybe displaying a validation error just under the throttle input when it is lesser than the check interval solves the problem? Or maybe @mdefazio could give us another idea :)

Reply:
I think @ersin-erdal and I discussed adding this check in a Zoom, if I remember correctly? Anyway, this is a check that we didn't already have even with rule-level throttle and notify, so maybe it's best to just remove it and open a new issue for it: https://github.com/elastic/kibana/issues/148569